nilearn/glm/second_level/second_level.py

```diff
@@ -60,6 +60,8 @@ def _check_second_level_input(
     second_level_input, design_matrix, confounds=None
 ):
     """Check second_level_input type."""
+    _check_design_matrix(design_matrix)
```

Comment:
This one was actually NOT used anywhere, so I added back where it should be run.